.gitignore

```diff
@@ -17,6 +17,7 @@
 /**/*.vc.db
 /.vscode/**
 /build/**
+*.sarif
```

Comment:
I am afraid that this will result in people not noticing that `*.sarif` files are created when they shouldn't be.

Reply:
I mean its just for git, and with my change the Errorlog property is just enabled in cloud builds...